src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/SelectExpandNode.cs

```diff
@@ -40,6 +43,7 @@ public SelectExpandNode()
         public SelectExpandNode(SelectExpandNode selectExpandNodeToCopy)
```

Comment:
>selectExpandNodeToCopy [](start = 49, length = 22)

We don't copy the "_selectItems and _edmProperty" ?

Reply:
They are consumed into the public properties once a selectexpandnode is initialized. We don't need to copy. 

---
In reply to: [262739928](https://github.com/OData/WebApi/pull/1738#discussion_r262739928) [](ancestors = 262739928)